galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.ipnetwork.manager/src/main/java/dev/galasa/ipnetwork/spi/SSHClient.java

```diff
@@ -613,7 +605,7 @@ public void run() {
 
                     long timeout = System.currentTimeMillis() - idleTimeout;
                     if (timeout >= lastCommandTimestamp) {
-                        logger.debug("No command issued after " + idleTimeout + " milliseconds, closing SSH session");
+                        logger.debug("SSH Client unused after " + idleTimeout + " milliseconds, freeing session");
```

Comment:
Similarly with these timeout/idleTimeout variables. They look like MilliSecs to me...